src/panels/NightSkyPanel/tabs/NightSkyMarkingsTab.tsx

```diff
@@ -0,0 +1,147 @@
+import { Button, Grid, Text } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+
+import { IconLabelButton } from '../components/IconLabelButton';
+
+export function NightSkyMarkingsTab() {
+  const openspace = useOpenSpaceApi();
+
+  return (
+    <>
+      <Button
+        mb={'lg'}
+        fullWidth
+        onClick={() => openspace?.action.triggerAction('os.nightsky.HideAllMarkings')}
+      >
+        HIDE ALL
+      </Button>
+      <Text my={'md'}>Alt/Az</Text>
+      <Grid columns={16} gutter={'sm'}>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Grid'} icon={'grid'} identifier={'AltAzGrid'} />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Meridian'}
+            icon={'line'}
+            identifier={'MeridianPlane'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Labels'}
+            icon={'text'}
+            identifier={'AltAzGridLabels'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Zenith'} icon={'dot'} identifier={'ZenithDot'} />
+        </Grid.Col>
+      </Grid>
```

Comment:
Use the `SimpleGrid` component here instead: 
```suggestion
      <SimpleGrid cols={4} spacing={'sm'}>
        <IconLabelButton title={'Grid'} icon={'grid'} identifier={'AltAzGrid'} />
        <IconLabelButton title={'Meridian'} icon={'line'} identifier={'MeridianPlane'} />
        <IconLabelButton title={'Labels'} icon={'text'} identifier={'AltAzGridLabels'} />
        <IconLabelButton title={'Zenith'} icon={'dot'} identifier={'ZenithDot'} />
      </SimpleGrid>
```
and the same for the grids below